week-1/Homework/mandatory/4-tax.js

```diff
@@ -17,7 +19,9 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function formatCurrency() {}
+function formatCurrency(productPrice) {
+  calculateSalesTax(productPrice)
```

Comment:
If the input of this function is 11:

Your function calls `calculateSalesTax` and returns the following
11 + (0.2 * 11) = **13.2**

The business has asked us to provide 2 decimal places and £0.00 format, so your function should return £13.20. Think about how you could implement this.